### ISSUE TITLE
Nuke ops loadout selection shouldn't happen on spawn

### ISSUE THREAD

**Issue by DamianX** [d3athrow/vgstation13#25223]
I played a round of nuke ops and as soon as I spawned I had to select my loadout, while we were discussing the plan.
I think you should instead spawn with a basic loadout by default and go to a machine/locker of some sort to choose your specialization, so that your UI isn't cluttered with the stupid menu while you do something else.

**Comment by DamianX** [comment]
After playing a second round, this time as team leader, it's even worse.
You get one prompt to set your "title", then you get the prompt for your loadout, then you get the prompt to set your last name???

**Comment by Blithering** [comment]
Agreed, when the team leader wants stealth, it's not good to have half your team be too busy picking grenade launcher. One of those loadout dispensers like ERT has would be nice
